content/docs/2.9/scalers/couchdb.md

```diff
@@ -0,0 +1,90 @@
++++
+title = "CouchDB"
+availability = "v2.9+"
+maintainer = "Community"
+description = "Scale applications based on CouchDB query results."
+go_file = "couchdb_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `couchdb` trigger that scales based on the outputs of a CouchDB query.
+
+```yaml
+triggers:
+- type: couchdb
+  metadata:
+    hostname: "test-release-svc-couchdb.couchdb-test-ns.svc.cluster.local"   
+    port: "5984" 
+    dbName: "animals" 
+    queryValue: "1" 
+    query: '{ "selector": { "feet": { "$gt": 0 } }, "fields": ["_id", "feet", "greeting"] }'
+    activationQueryValue: "1"
+    metricName: "global-metric"
```

Comment:
Please remove `metricName` property, we try to get rid of those from user config, since it is internal thing only. Thanks!